src/battle-searchresults.tsx

```diff
@@ -8,7 +8,7 @@
  */
 
 class PSSearchResults extends preact.Component<{search: DexSearch}> {
-	readonly URL_ROOT = '//dex.pokemonshowdown.com/';
+	readonly URL_ROOT = `//${Config.routes.dex}/`;
```

Comment:
`Config` is not defined here, either.

Reply:
On reflection, I think we can allow this. Including `config` in PSdex is probably better than rewriting this.